sorc/ncep_post.fd/MISCLN.f

```diff
@@ -2212,10 +2217,11 @@ SUBROUTINE MISCLN
             CALL CALLCL(PBND(1,jsta,1),TBND(1,jsta,1),          &
                         QBND(1,jsta,1),EGRID1,EGRID2)
             IF (IGET(109)>0) THEN
+               GRID1=spval
 !$omp parallel do private(i,j)
                DO J=JSTA,JEND
                  DO I=1,IM
-                   GRID1(I,J) = EGRID2(I,J)
+                   IF(T1D(I,J) < spval) GRID1(I,J) = EGRID2(I,J)
```

Comment:
line 2224, should it be:
IF(EGRID2(I,J) < spval) GRID1(I,J) = EGRID2(I,J) ?

Reply:
will do

Reply:
done